### ISSUE TITLE
Layout problem Switch, Weather and so on

### ISSUE THREAD

**Issue by chatainsim** [EdddieN/machinon-domoticz_theme#27]
Hi,
I have an issue with sensor layout in Firefox:
![capture](https://user-images.githubusercontent.com/1167931/43046624-f56b1528-8dcb-11e8-8a85-e15d94e8e1c7.PNG)
![capture1](https://user-images.githubusercontent.com/1167931/43046627-fccb8ca8-8dcb-11e8-9deb-d15a06a88873.PNG)

Chrome is not (less) impacted:
![image](https://user-images.githubusercontent.com/1167931/43046651-50606b5e-8dcc-11e8-9954-86c12555f19a.png)
![image](https://user-images.githubusercontent.com/1167931/43046654-5a4d25b2-8dcc-11e8-8d73-59db2a502f6b.png)

**Comment by landaisbenj** [comment]
Thanks for your report. We will see that.

**Comment by DewGew** [comment]
Cache problems in firefox. Better to use chrome or edge.

**Comment by DewGew** [comment]
Is there a reason why temeperatures, weather and utillity names has bigger font size (20px) than dashboard and devices (16px). I think it looks better if the names has same font size.

**Comment by EdddieN** [comment]
No, maybe something has slipped through... should all have the same font

**Comment by EdddieN** [comment]
Which font are you using? I seem to get a very ugly one on my version. Is the font included with the theme?

**Comment by EdddieN** [comment]
Just to confirm, these are the fonts types:
Dashboard
![screen shot 2018-08-31 at 17 24 40](https://user-images.githubusercontent.com/39434245/44924328-dd79e480-ad42-11e8-8bd2-d700403595b0.png)

Rest of tabs
![screen shot 2018-08-31 at 17 24 22](https://user-images.githubusercontent.com/39434245/44924340-e66ab600-ad42-11e8-9993-531230e569fc.png)
![screen shot 2018-08-31 at 17 24 08](https://user-images.githubusercontent.com/39434245/44924344-eb2f6a00-ad42-11e8-9d71-371cf551d4fa.png)

**Comment by DewGew** [comment]
#45
